### ISSUE TITLE
Proposed Analysis: Molecularly subtype non-MB/ATRT embryonal tumors

### ISSUE THREAD

**Issue by jharenza** [AlexsLemonade/OpenPBTA-analysis#251]
#### Scientific goals

_What are the scientific goals of the analysis?_
1) Subtype non-MB embryonal tumors (non-medulloblastoma/ATRT; eg: old PNET-classified tumors) into ETMRs and CNS embryonal NOS. 
2) Assess whether any other tumors may have been mis-diagnosed but fit into these categories (eg: ETMR characteristics in other tumor diagnoses would make these newly ETMR cases).
3) Render the results in tabular form in a notebook.

#### Proposed methods

_What methods do you plan to use to accomplish the scientific goals?_
1) Review of copy number, mutation, and expression data.
2) Render results in tabular form in a notebook.

**ETMR, C19MC-altered**
- Embryonal tumor with multilayered rosettes (ETMR), C19MC-altered
- These tumors have focal amplification of the miRNA cluster on chr19 (denoted C19MC) and often have gene fusions involving _TTYH1_ and chr19 miRNA cluster genes
- These tumors have high expression of _LIN28A_ and serves as a biomarker for ETMRs

**ETMR, NOS**
- Embryonal tumor with multilayered rosettes (ETMR)
- These tumors have high expression of _LIN28A_ and serves as a biomarker for ETMRs, but do not show focal amplification of C19MC.

**CNS HGNET-MN1**
- CNS high-grade neuroepithelial tumor with MN1 alteration
- Likely previously diagnosed as PNET.
- Contain gene fusions involving 5' _MN1_. 3' fusion partners can include _BEND2_ and _CXXC5_. 
- Predominantly female patients.

**CNS HGNET-BCOR**
- CNS high-grade neuroepithelial tumor with BCOR alteration
- Tumors have internal tandem duplication of _BCOR_
- Median age of diagnosis less than 10 years

**CNS NB-FOXR2**
- Central nervous system (CNS) neuroblastoma with FOXR2 activation
- Over-expression and/or gene fusions in _FOXR2_

**CNS EFT-CIC**
- CNS Ewing sarcoma family tumor with CIC alteration
- Alterations in _CIC_, commonly fused with _ NUTM1_

**CNS Embryonal, NOS**
- CNS Embryonal tumor, not otherwise specified
- Tumors previously called PNET that do not fit into other groups above.

#### Required input data
_What input data will you use for this analysis?_
Copy number, RNA expression data, histologies file.

#### Proposed timeline
_What is the timeline for the analysis?_
1 week

#### Relevant literature

_If there is relevant scientific literature, put links to those items here._
[Link](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6315657/) to Embryonal Tumors of the Central Nervous System in Children: The Era of Targeted Therapeutics
[Link](https://www.cancer.gov/types/brain/hp/child-cns-embryonal-treatment-pdq) to NCI PDQ.
[Link](https://www.ncbi.nlm.nih.gov/pubmed/28744687) to LIN28A, a sensitive immunohistochemical marker for Embryonal Tumor with Multilayered Rosettes (ETMR), is also positive in a subset of Atypical Teratoid/Rhabdoid Tumor (AT/RT).

**Comment by sjspielman** [comment]
Will start this in the next couple of days after familiarizing myself with existing molecular subtyping analyses. Stay tuned, team.

**Comment by sjspielman** [comment]
Hi @jharenza , I wonder if you could clarify the scope of this analysis further - based on my reading of the PR, you would like to include *any sample* not previously identified as having MB or ATRT histology, but I wonder if I am reading this correctly or if I need some more context. For example, there is no PNET annotation within the histology file, so understanding which non-MB/ATRT samples to focus on for initial molecular subtyping (novel discovery and/or identification of incorrect annotations will come second) isn't immediately clear - unless this PR really does mean everything else, in which case I'm good to go. Thanks for any clarification about the scope here!

**Comment by jharenza** [comment]
Hi @sjspielman! First, thanks for working on this. Second, looks like I was inconsistent with the `broad_histology` so if you subset for just `Embryonal tumor` you will not see PNETs, but if you do something like `emb <- subset(clin, grepl("Embryonal tumor", clin$broad_histology))`, then you will see those PNETs in `disease_type_old`, and a few more. 

`as.data.frame(table(emb$disease_type_old))`:

| Var1 | Freq |
|-----------------------------------------|------|
| Atypical Teratoid Rhabdoid Tumor (ATRT) | 60 |
| Ganglioneuroblastoma | 6 |
| Medulloblastoma | 229 |
| Other | 7 |
| Supratentorial or Spinal Cord PNET | 34 |

The ganglioneuroblastoma, other, and PNET should be the samples subtyped. For the `Other` histologies, I had used our "database" to add a `disease_type_new`, but those were from pathology and we should re-confirm them. Hope this helps :)!

**Comment by sjspielman** [comment]
Thanks for the clarification, @jharenza! I see this - so initial analysis will consist of 47 (6+7+34) known samples, yes?

**Comment by jharenza** [comment]
Correct!

**Comment by sjspielman** [comment]
Getting back to this analysis after cleaning up the GSVA scores - I'm wondering if there's an issue with some of the data or not. Of the 47 samples to be subtyped, only 24 samples appear to have corresponding expression data, as in:
```
expression_file <- file.path(data_dir, "pbta-gene-expression-rsem-fpkm.stranded.rds") 
metadata_file <- file.path(data_dir, "pbta-histologies.tsv")
metadata   <- readr::read_tsv(metadata_file)
expression <- readr::read_rds(expression_file)

## Subset metadata to the samples that need subtyping
metadata %>% 
  filter(str_detect(broad_histology, "Embryonal tumor"),  ## Keep all embryonal 
         disease_type_old != "Medulloblastoma",           ## discard MB (this line) and ATRT (next line)
         disease_type_old != "Atypical Teratoid Rhabdoid Tumor (ATRT)") -> embryonal_unclassified

## Which of these samples have RNA expression data?
embryonal_unclassified$Kids_First_Biospecimen_ID %in% names(expression)
 [1]  TRUE  TRUE  TRUE  TRUE FALSE  TRUE FALSE FALSE  TRUE  TRUE  TRUE FALSE  TRUE FALSE  TRUE FALSE FALSE
[18] FALSE  TRUE  TRUE  TRUE  TRUE  TRUE FALSE FALSE FALSE FALSE FALSE FALSE FALSE  TRUE  TRUE FALSE  TRUE
[35]  TRUE FALSE  TRUE  TRUE FALSE FALSE  TRUE FALSE  TRUE FALSE FALSE  TRUE FALSE

sum(embryonal_unclassified$Kids_First_Biospecimen_ID %in% names(expression))
[1] 24

```

Without having all expression data, it's not clear to me how one can subtype using expression data. @jharenza, do you have any particular insights here? I've confirmed my data is up to date with release v12.

**Comment by jaclyn-taroni** [comment]
@sjspielman it doesn't look like you've filtered the `metadata` based on the `experimental_strategy` -- so there will be WGS or WXS samples in `embryonal_unclassified` which will be missing from the expression data.

You can find DNA-seq, RNA-seq _pairs_ using the `sample_id` column. Each _library_ has its own biospecimen ID. The `sample_id` is what captures the same event or sample. This might provide some needed context: https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/260#issuecomment-552927982 and here's what we ended up doing in the OncoPrint case: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/5431b72d37d4d55a09bb63b5d8f6abfce5b4309f/analyses/oncoprint-landscape/00-map-to-sample_id.R

**Comment by jaclyn-taroni** [comment]
Side note: This issue has tripped up multiple people including me, so we should probably document this!

**Comment by sjspielman** [comment]
Aha, of course thank you!! This brings the sample size down to 29 (not 47), with 4 of those having polyA expression and 25 stranded. I would not imagine it's ok to "merge" different RNAseq selection strategies so for now I'll work with 25 stranded selection ones only.

**Comment by jharenza** [comment]
@sjspielman - I had reached out to an ETMR domain expert, Derek Hanson, via email and am posting the exchange below for tracking purposes:

> Hi Derek,
>  
> Nice to e-meet you. One of our major efforts within the OpenPBTA is molecularly subtyping all of the brain tumors. Specifically for non-MB/non-ATRT embryonal tumors, I created this ticket in GitHub (where all analyses are being tracked). Stephanie is the analyst currently working on the subtyping (pull request [here](#401 )) and I have read through some of her initial analyses and offered some guidance, but would be great to get another set of eyes on this to be sure I am directing her properly.
>  
> Essentially, we are starting with ETMRs since they have the easier molecular hallmarks (see [this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/401#issuecomment-570961443)). We are approaching this by:
> 1)      First assessing whether any PBTA tumors contain TTYH1 fusions that lie within the C19MC region and if these same tumors have high LIN28A expression (biomarker). If yes, we are classifying as ETMRs. We also have WGS on some (not all) and I am spot-checking those for focal C19MC amplification, which is a give-away for ETMRs.
> a.       A few questions came to mind – we see other non-canonical, if you will, fusions, like TTYH1-DPRX/RNU-698P (the 5’ fusion partner here is two genes because the breakpoint is intergenic between those two). Upon a UCSC search, you can see that these genes are adjacent the miRNA cluster, so I am considering this as a potential ETMR and we will validate this via LIN28A expression. Is this valid, in your opinion? Do you see/report those fusions as well?
>                                                               i.      There are also samples with other TTYH1 fusions, but these are not within the C19MC region, so these would not be considered ETMRs off the bat.
>                                                             ii.      I am making an assumption that this is the case, but as a double check, would you also say high LIN28A expression is required for ETMR classification?
> 2)      Next, we will subtype all non-MB/non-ATRTs into ETMRs or other embryonal tumors as described in the first ticket. Do you see anything in that ticket we should add/delete/modify?
> 
> If you have a GitHub username, you can comment directly on any of these tickets (if not, you can quickly create a username and comment), else I will just paste your email responses there. Once we have a finalized list of tumors, features, and classifications, would also love to get this back to you for confirmation!

and his response:

> Hi Jo Lynne,
>  
> I took some time to review your email and the information in GitHub.
>  
> I think that the approach to look for TTYH1 fusions within the C19MC region along with LIN28A expression is valid, as well as looking for focal C19MC amplification.
>  
> I have not personally seen cases where the fusion has been adjacent to the miRNA cluster, but these tumors are very rare and I have only been able to review the molecular data on a handful of cases.  This paper by Nada Jabado’s lab is probably the best reference when it comes to TTHY/C19MC fusions and may shed some more light on the situation - https://www.nature.com/articles/ng.2849.    I think considering these cases with an adjacent fusion as potential ETMR and further validating with LIN28A expression is a reasonable plan. 
>  
> LIN28A appears to be overexpressed in 100% of ETMRs and is highly specific for the tumor with only about 10% of ATRTs also expressing LIN28A - https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3508282/.  I have never seen an ETMR case reported in any series where LIN28A was not overexpressed. 
>  
> You may also want to consider DNMT3B expression as another method for validation.  According to the Jabado paper above, this is also highly specific for ETMR among brain tumors. 
>  
>  
> In summary, any tumor with TTHY1/C19MC fusion or C19MC amplification should be easy to classify.  For the ~10% of ETMRs that do not have C19MC amplification, LIN28A and/or DNMT3B overexpression would be highly suggestive of ETMR.  These overexpressions could also be confirmatory for those tumors with the adjacent fusions, which I agree are most likely ETMR.   
>  
>  
> I think that the plan outlined in the first GitHub ticket for classifying ETMR and other embryonal tumors is reasonable.  As I mentioned, analyzing LIN28A expression by itself is likely sufficient for classifying ETMRs, but DNMT3B could also be used to confirm if there were any questions.
>  
>  
> Please let me know if you have any questions or if there is anything else that I can do to be of help.
>  
> Thanks,
> Derek

**Comment by jaclyn-taroni** [comment]
I am planning on working on this over the next few days.

**Comment by jaclyn-taroni** [comment]
[`molecular-subtyping-embryonal`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-embryonal) will need to be rerun with annotated focal consensus calls (#186) and new GISTIC calls (#453).

**Comment by jaclyn-taroni** [comment]
Seems like [Sturm et al.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5139621/) is good thing to link here -- the NCI PDQ section on CNS EFT-CIC cites it.

**Comment by jaclyn-taroni** [comment]
Closed via #458 - the subtype labels will need to go into a data release.
